CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Added prefix to theme-aggregator
+### Changed
+* Removed blank lines from files
```

Comment:
We don't need to add a change log for removing blank lines since this won't affect how the content released in terra-toolkit will run for consumers

Reply:
https://github.com/cerner/terra-toolkit/pull/367/commits/e54b18556150cb4bdb343d1fb5d7338d6d60200f